pypeit/spectrographs/gemini_gmos.py

```diff
@@ -818,9 +835,33 @@ def get_detector_par(self, det, hdu=None):
             nonlinear       = 0.95,
             mincounts       = -1e10,
             numamplifiers   = 4,
-            gain            = np.atleast_1d([1.83]*4),
-            ronoise         = np.atleast_1d([3.98]*4),
+            gain            = np.atleast_1d([1.74]*4),  # Slow Readout,	Low Gain
+            ronoise         = np.atleast_1d([3.75]*4),  # Slow Readout,	Low Gain
             )
+
+        # account for the CCD upgrade happened on 2023-12-14
+        if hdu is not None:
```

Comment:
Should this perhaps be defined before the detector dictionaries? (i.e. define parameters `gain1`, `gain2`, etc...). I think it's fine to update, but might be cleaner if we set variables near the top of this function and use those variables in the dictionary.

Reply:
I do actually prefer to leave it like this. Other spectrographs have the same flow. Also, the code below is only run if `hdu` is provided, which would make the code full of `if...else` if I define the parameters before the dictionary.